cypress/integration/seededFlows/adminFlows/users/userIndexView.spec.js

```diff
@@ -4,9 +4,9 @@ describe('User index view', () => {
       cy.testSetup();
       cy.fixture('users/adminUser.json').as('user');
       cy.get('@user').then((user) => {
-        cy.loginUser(user)
-          .then(() => cy.enableFeatureFlag('member_index_view'))
-          .then(() => cy.visitAndWaitForUserSideEffects('/admin/users'));
+        cy.loginUser(user).then(() =>
+          cy.visitAndWaitForUserSideEffects('/admin/users'),
+        );
```

Comment:
```suggestion
        cy.loginAndVisit(user, '/admin/users');
```
(not blocking) - we could take advantage of this simpler helper now if we wanted to 🙂